x/auth/tx/generator.go

```diff
@@ -35,6 +38,15 @@ func (g generator) NewTxBuilder() client.TxBuilder {
 	return newBuilder(g.marshaler, g.pubkeyCodec)
 }
 
+func (g generator) WrapTxBuilder(newTx sdk.Tx) (client.TxBuilder, error) {
+	newBuilder, ok := newTx.(client.TxBuilder)
```

Comment:
```suggestion
	newBuilder, ok := newTx.(*builder)
```

I think this actually should expect `*builder` as that's what we're using with this config.

Reply:
Ah yes, I updated it for StdTx. Missed it here, Thanks